.github/workflows/on_release_build_contracts.yml

```diff
@@ -9,7 +9,7 @@ permissions:
 
 jobs:
   build:
-    uses: multiversx/mx-sc-actions/.github/workflows/reproducible-build.yml@v2.2.1
+    uses: multiversx/mx-sc-actions/.github/workflows/reproducible-build.yml@v2.3.0
     with:
-      image_tag: v4.1.1
+      image_tag: next
```

Comment:
Will be adjusted after publishing a tag for the new Docker image for reproducible builds.